src/Compilers/CSharp/CSharpAnalyzerDriver/CSharpDeclarationComputer.cs

```diff
@@ -82,46 +82,54 @@ private static bool InvalidLevel(int? level)
                     {
                         var t = (TypeDeclarationSyntax)node;
                         foreach (var decl in t.Members) ComputeDeclarations(model, decl, shouldSkip, getSymbol, builder, newLevel, cancellationToken);
-                        builder.Add(GetDeclarationInfo(model, node, getSymbol, cancellationToken));
+                        var attributes = GetAttributes(t);
```

Comment:
>var attributes = GetAttributes(t); [](start = 24, length = 34)

What about attributes on type parameters?

Reply:
Tracked with [https://github.com/dotnet/roslyn/issues/23348](https://github.com/dotnet/roslyn/issues/23348)

---
In reply to: [152411734](https://github.com/dotnet/roslyn/pull/23318#discussion_r152411734) [](ancestors = 152411734)

Reply:
Actually, I will address type parameters and return type attributes in this PR itself.

---
In reply to: [152423846](https://github.com/dotnet/roslyn/pull/23318#discussion_r152423846) [](ancestors = 152423846,152411734)